src/extension.ts

```diff
@@ -3,6 +3,8 @@ import { join } from 'path';
 import * as vscode from 'vscode';
 import { parseTestName, platformWin32, quote, slash } from './util';
 
+var globalCommand;
```

Comment:
shouldn't this be a "let" instead of "var"? as far as i know "var" is deprecated. also i was a bit confused about the name "gobalCommand" because acutally this variable holds the "previousCommand"

Reply:
@firsttris thanks for checking out. Don't think var is deprecated but agree should be let. Also changed the variable name!